buyer_data/models.py

```
@@ -1,7 +1,8 @@
-from django.db import models
+from django.db import models, transaction
+from django.dispatch import receiver
 from rest_framework.exceptions import ValidationError
 from common.models import CustomUser, UserExtensionMixin, VerificationMixin
-from estate_data.models import Estate
+from estate_data.models import Estate, EstateMetrics
```

Ruff (0.11.9)

`estate_data.models.EstateMetrics` imported but unused

Remove unused import: `estate_data.models.EstateMetrics`

(F401)